assets/src/globals.js

```diff
@@ -22,6 +22,8 @@ const user = Object.freeze({
   LTIIsCourseDataLoaded: mylaGlobals.lti_is_course_data_loaded
 })
 
+const viewHelpURLs = Object.freeze(mylaGlobals.view_help_urls)
```

Comment:
I don't feel Object.freeze is strictly needed since these are static's and will have less effect of changing, and if somebody will change.

Reply:
I hadn't used this before and was following the example  of mylaglobals.  Is there a downside to using it? I can remove it for sure.